### PR TITLE
When rotating secrets, check for IAM username without account ID suffix label at the end

### PR DESCRIPTION
**Pull request by AlexVulaj**
As part of investigating https://issues.redhat.com/browse/OHSS-21052, we determined that osdctl was failing to rotate secrets because older clusters do not have the account ID suffix labels appended at the end of IAM usernames. This fix will have osdctl check to see if the IAM username is present without the label, and then check again with the label upon failure.

### CODE REVIEW

**Review by mjlshen — APPROVED** [review]
/approve

File: cmd/account/rotate-secret.go [hunk 1]
```diff
@@ -205,14 +205,17 @@ func (o *rotateSecretOptions) run() error {
...
 	// Username is osdManagedAdmin-aaabbb
-	osdManagedAdminUsername := common.OSDManagedAdminIAM + "-" + accountIDSuffixLabel
+	osdManagedAdminUsername := common.OSDManagedAdminIAM
 
 	// Create new access key
-	createAccessKeyOutput, err := awsClient.CreateAccessKey(&iam.CreateAccessKeyInput{
-		UserName: aws.String(osdManagedAdminUsername),
-	})
+	createAccessKeyOutput, err := awsClient.CreateAccessKey(&iam.CreateAccessKeyInput{UserName: aws.String(osdManagedAdminUsername)})
 	if err != nil {
```
Comment (wshearn) [inline cmd/account/rotate-secret.go:212]: instead of checking if not nil here, can we check if it is the user not found error?

Comment (AlexVulaj) [reply cmd/account/rotate-secret.go:213]: Added a check below.

**Comment by iamkirkbater** [comment]
Was this able to have been tested on a cluster in staging?

**Comment by AlexVulaj** [comment]
@iamkirkbater Yes, this was tested on a staging cluster for both the new naming convention (`osdManagedAdmin-abcdef`) and the older convention (`osdManagedAdmin`).

**Pull request merged**